apps/web/package.json

```diff
@@ -38,13 +38,14 @@
     "lucia": "^3.2.0",
     "monaco-editor": "^0.50.0",
     "nanoid": "^5.0.7",
-    "next": "^14.3.0-canary.87",
+    "next": "15.0.0-canary.168",
     "next-themes": "^0.3.0",
     "nextjs-toploader": "^1.6.12",
     "nprogress": "^0.2.0",
+    "oslo": "1.2.0",
```

Comment:
was this a missing dependency?

Reply:
Yeah Turbopack actively shows that the dependency is missing when it's external, it would fail at runtime currently without this being installed directly.